packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -104,7 +98,11 @@ export class InteractionManager extends BaseManager<
     private touchDown = false;
     private dragStartElement?: HTMLElement;
 
-    private pausers: Record<PauseType, number> = { animation: 0, 'context-menu': 0 };
+    private stateQueue: Set<InteractionState> = new Set();
+    private _state: InteractionState = InteractionState.Default;
+    public get state() {
```

Comment:
nit: I personally try to avoid unnecessary setters and getters.
I think we can simply expose this property, it's internal anyhow, or maybe just replace it with a method `getState` and keep the private property named `state` and avoid the underscore prefix and duplicated definition.

Reply:
I have no strong opinions about this. I would however prefer making `_state` public than having a trivial `getState()` method.